eng/pipelines/docs.yml

```diff
@@ -34,20 +34,13 @@ jobs:
         displayName: "Install typedoc"
 
       - script: |
-          npm install nunjucks
-        displayName: "Install nunjucks"
-
-      - script: |
-          npm install fs-extra
-        displayName: "Install fs-extra"
-
-      - script: |
-          npm install yargs
-        displayName: "Install yargs"
+          npm install
+        workingDirectory: $(System.DefaultWorkingDirectory)/eng/tools/generate-doc
+        displayName: "Install tool dependencies"
 
       - pwsh: |
           cd $(Build.SourcesDirectory)
-          node .\common\scripts\generate-doc.js --dgOp "dg" $(additionalArgs)
+          node .\eng\tools\generate-doc\index.js --dgOp "dg" $(additionalArgs)
```

Comment:
We can still build the docs in the old methodology correct?

Reply:
Yes we can. I need to add another arg `--oldIndex` and pass empty quotes with the exclude list